apps/console/src/index.jsp

```diff
@@ -16,36 +16,72 @@
-        <meta name="referrer" content="no-referrer" />
-
-        <link href="<%= htmlWebpackPlugin.options.publicPath %>/libs/themes/default/theme.<%= htmlWebpackPlugin.options.themeHash %>.min.css" rel="stylesheet" type="text/css"/>
-        <link rel="shortcut icon" href="<%= htmlWebpackPlugin.options.publicPath %>/libs/themes/default/assets/images/branding/favicon.ico" />
-        <script>
-            var contextPathGlobal = "<%= htmlWebpackPlugin.options.publicPath %>";
-            var serverOriginGlobal = "<%= htmlWebpackPlugin.options.serverUrl %>";
-            var superTenantGlobal = "<%= htmlWebpackPlugin.options.superTenantConstant %>";
-            var tenantPrefixGlobal = "<%= htmlWebpackPlugin.options.tenantPrefix %>";
-        </script>
+        <script src="https://unpkg.com/@asgardeo/auth-spa@latest/dist/asgardeo-spa.production.min.js"></script>
     </head>
     <body>
-        <noscript>
-            You need to enable JavaScript to run this app.
-        </noscript>
-        <div id="root"></div>
+        <script>
+            var userAccessedPath = window.location.href;
+            var applicationDomain = window.location.origin;
+            var userTenant = userAccessedPath.split("/t/")[1] ?  userAccessedPath.split("/t/")[1].split("/")[0] : null;
+
+            var serverOrigin = "<%=serverUrl%>";
+
+            function getApiPath(path) {
+                if(path) {
+                    return serverOrigin + path;
+                }
+
+                return serverOrigin;
+            }
+
+            var auth = AsgardeoAuth.AsgardeoSPAClient.getInstance();
+
+            var authConfig = {
+                signInRedirectURL: applicationDomain.replace(/\/+$/, ''),
+                signOutRedirectURL: applicationDomain.replace(/\/+$/, ''),
+                clientID: "CONSOLE",
+                serverOrigin: getApiPath(),
+                responseMode: "form_post",
+                scope: ["openid SYSTEM"],
+                storage: "webWorker",
+                endpoints: {
+                    authorizationEndpoint: getApiPath(userTenant ? "/t/a/oauth2/authorize?ut="+userTenant.replace(/\/+$/, '') : "/t/a/oauth2/authorize"),
+                    clockTolerance: 300,
+                    jwksEndpointURL: undefined,
+                    logoutEndpointURL: getApiPath("/t/a/oidc/logout"),
+                    oidcSessionIFrameEndpointURL: getApiPath("/t/a/oidc/checksession"),
+                    serverOrigin: getApiPath(),
+                    tokenEndpointURL: undefined,
+                    tokenRevocationEndpointURL: undefined,
+                    wellKnownEndpointURL: undefined,
+                },
+                enablePKCE: true,
+                overrideWellEndpointConfig: true
+            }
+
+            auth.initialize(authConfig);
+
+            auth.trySignInSilently().then(res => {
+                if(res === false) {
```

Comment:
Is there a possibility of res being any other falsy value than `false`?
```suggestion
                if(res === false) {
```

Reply:
No the response can be boolean or user info object
References: https://github.com/asgardeo/asgardeo-auth-spa-sdk/blob/5d60a661508ab749aa0a6ad08a7505656f2f7605/lib/src/models/client.ts#L65